packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -710,4 +716,49 @@ contract OPContractsManager is ISemver, Initializable {
     function blueprints() public view returns (Blueprints memory) {
         return blueprint;
     }
+
+    /// @notice Returns L1ERC721BridgeImpl implementation contract address.
+    function getL1ERC721BridgeImpl() public view returns (address) {
+        return implementationContracts.l1ERC721BridgeImpl;
+    }
```

Comment:
Why did we add all these getter methods here? Unclear if this was required for the diff in this PR or if this is a separate refactor. It looks like these are all just trivial aliases and the functions are only used once, so perhaps worth deleting?